src/Utilities/CleverDicts.jl

```diff
@@ -148,13 +148,6 @@ function Base.haskey(c::CleverDict{K}, key::K) where {K}
     return haskey(c.dict, key)
 end
 
-function Base.keys(c::CleverDict{K}) where {K}
```

Comment:
So what method does this fallback to now?

Reply:
```julia
julia> @which keys(d)
keys(a::AbstractDict) in Base at abstractdict.jl:103
```

Reply:
The julia Base KeySet because CleverDict is an AbstractDict.
It will ultimately use the CleverDict iterator and just return the first element.